pages/tutorials.js

```diff
@@ -17,6 +17,11 @@ const Hero = styled.div`
   align-items: center;
   padding-left: 10%;
   padding-right: 10%;
+  background-image: url("/images/backgrounds/tutorials.svg");
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: relative;
```

Comment:
Consider `fixed`

Reply:
> Consider `fixed`

Done, thank you! It can be really nice, especially into the Home Page :)